vector/src/main/java/im/vector/app/features/home/HomeActivity.kt

```diff
@@ -138,6 +139,8 @@ class HomeActivity :
     @Inject lateinit var fcmHelper: FcmHelper
     @Inject lateinit var nightlyProxy: NightlyProxy
 
+    private var isNewAppLayoutEnabled: Boolean = false // delete once old app layout is removed
```

Comment:
NIT (merge without addressing): better reference a line of code or issue, so it will be possible to check this condition. Here it could be an app layout flag